dispatcher/queue.go

```diff
@@ -1,13 +1,44 @@
 package dispatcher
 
 import (
-	"container/list"
+	"container/heap"
 	"sync"
 )
 
-// A PriorityQueue implements heap.Interface and holds Items.
+type Jobs struct {
+	list []*Job
+}
```

Comment:
```suggestion
type Jobs []*Job
```

Reply:
Do we need to export this type?
Could we just make this `jobs`?

Reply:
Done in https://github.com/newscred/webhook-broker/pull/134/commits/c2ce21c6040245a0109ff1ad1d9111f5a0b0760d